docs/ref/django-admin.txt

```diff
@@ -240,6 +240,15 @@ are prefixed by ``"###"``.
 The settings module to compare the current settings against. Leave empty to
 compare against Django's default settings.
 
+.. django-admin-option:: --output
+
+.. versionadded:: 2.0
+
+Specifies the output format. Available values are ``hash`` and
+``unified``. Default is ``hash``. The ``hash`` mode displays the output similar
+to ``"postconf -n"`` while the ``unified`` mode displays the output similar to
```

Comment:
I'm not sure what "postconf -n" does, and I'm not sure if most other readers will know either. Testing it out, I don't see how the output is similar. Perhaps it would be better to describe it in words.

Release notes for `docs/releases/2.0.txt` could also be added.